Dockerfile.ppc64le

```diff
@@ -0,0 +1,100 @@
+# This file describes the standard way to build Docker, using docker
+#
+# Usage:
+#
+# # Assemble the full dev environment. This is slow the first time.
+# docker build -t docker -f Dockerfile.gccgo .
```

Comment:
This should be `.ppc64le` and should mention that this will _only_ work on a `ppc64le` machine.